packages/gafl-webapp-service/src/handlers/page-handler.js

```diff
@@ -25,6 +25,22 @@ const displayAnalytics = request => {
   return request.path.startsWith('/buy')
 }
 
+const omitPageFromAnalytics = async request => {
+  const analytics = await request.cache().helpers.analytics.get()
+
+  if (
+    analytics &&
+    analytics[ANALYTICS.pageSkipped] !== true &&
+    analytics[ANALYTICS.seenMessage] &&
+    analytics[ANALYTICS.omitPageFromAnalytics] !== true
+  ) {
+    await request.cache().helpers.analytics.set({ [ANALYTICS.omitPageFromAnalytics]: true, [ANALYTICS.pageSkipped]: true })
+    return
```

Comment:
You shouldn't be returning nothing. You need to restructure the function to have an `else` rather than doing an empty return here.